.github/workflows/run-e2e-tests.yaml

```
@@ -40,48 +40,48 @@ jobs:
       id-token: write
     steps:
       - name: Get actions
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: dispatch to terra-github-workflows
         id: FirstAttemptCreateBee
         continue-on-error: true
         uses: ./.github/actions/create-bee
         with:
-            bee_name: ${{ env.BEE_NAME }}
-            token: '${{ env.TOKEN }}'
+          bee_name: ${{ env.BEE_NAME }}
+          token: '${{ env.TOKEN }}'
 
       - name: retry dispatch to terra-github-workflows in case of failure
         if: steps.FirstAttemptCreateBee.outcome == 'failure'
         uses: ./.github/actions/create-bee
         with:
-            bee_name: ${{ env.BEE_NAME }},
-            token: ${{ env.TOKEN }}
+          bee_name: ${{ env.BEE_NAME }},
+          token: ${{ env.TOKEN }}
 
   attach-landing-zone-to-bee-workflow:
     runs-on: ubuntu-latest
     needs: [ init-github-context, create-bee-workflow, params-gen ]
     steps:
       - name: Get actions
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: dispatch to terra-github-workflows
         id: FirstAttemptLandingZone
         continue-on-error: true
         uses: ./.github/actions/landing-zone-attach
         with:
-            run_name: "${{ env.ATTACH_BP_TO_LZ_RUN_NAME }}"
-            bee_name: ${{ env.BEE_NAME }}
-            token: '${{ env.TOKEN }}'
-            project_name: ${{needs.params-gen.outputs.project-name }}
+          run_name: "${{ env.ATTACH_BP_TO_LZ_RUN_NAME }}"
+          bee_name: ${{ env.BEE_NAME }}
+          token: '${{ env.TOKEN }}'
+          project_name: ${{needs.params-gen.outputs.project-name }}
```

Super nit, but as long as you're formatting...

```suggestion
          project_name: ${{ needs.params-gen.outputs.project-name }}
```

Same below

---

fixed